### PR TITLE
fix(nf-vc): changed signature of NfValueConverter

### PR DESCRIPTION
**Pull request by AdamWillden**
This update is proposed to bring it in line with the number formatter which addressed two issues (#100 & #101)

I'm not sure what the number formatter does with null or empty values but I think it makes sense to copy the behaviour from the date formatter in this case.

### CODE REVIEW

**Comment by AdamWillden** [comment]
Ahh! Bad colon - bare with me

File: src/nf.js [hunk 1]
```diff
@@ -6,8 +6,21 @@ export class NfValueConverter {
...
+      || typeof value === 'undefined'
+      || (typeof value === 'string' && value.trim() === '')
+      ) {
+      return value;
+    }
+
+    if (nfOrOptions && (typeof nfOrOptions.format === 'function')) {
+      return nfOrOptions.format(value);
+    } else if (nf) {
+      console.warn('This ValueConverter signature is depcrecated and will be removed in future releases. Please use the signature [nfOrOptions, locale]'); // eslint-disable-line no-console
```
Comment (atsu85) [inline src/nf.js]: use aurelia logger instead of direclty using console

Comment (AdamWillden) [inline src/nf.js]: I've simply copied the date time value converter code which was implemented by an aurelia developer. I even questioned how they wanted it done [here](https://github.com/aurelia/i18n/issues/100#issuecomment-213424746).

I propose if this needs changing it is done in a separate PR and both are updated.

**Comment by zewa666** [comment]
@AdamWillden I think this sounds reasonable, I've just added some more tests to it.
Regarding console warnings, @atsu85 is right and we should use the aurelia logger. It was my mistake to use console.warn directly inside the dfvc. Sry about that, fixed it in your PR as well.

Again thanks for the great contribution guys :+1: :+1:
